README.md

```diff
@@ -22,3 +22,36 @@ In some cases, we need to make changes to those to get a working build.
 `.github/workflows/pr-if-new-kernel.yml` contains the workflow to create new PRs based on `./update-kernel.py` if new patch versions of the LTS kernel are available.
 
 `.github/workflows/build.yml` contains the workflow to build and release the kernel binaries.
+
+
+## Backports 
+
+The main branch is always on latest Long term supported Linux version defined at kernel.org. 
+
+We maintain also older supported kernel versions, if they are required by supported Garden Linux versions.
+To see what Garden Linux versions are currently supported, please check out the [Active and Next Release](https://github.com/gardenlinux/gardenlinux?tab=readme-ov-file#active-and-next-releases) section in gardenlinux/gardenlinux repository. 
+
+Any kernel version that we need to maintain other than the latest LTS in main, are maintained in `maint-<MAJOR.MINOR>` branches (e.g. maint-6.6).
+Actual backport releases need to branch of from the respective `maint-<MAJOR.MINOR>` branch and include the corresponding `.container` file for target backport.
+
+Branches containing the `.container` file must be named according to the `rel-MAJOR` naming scheme (e.g. rel-1443).
+
+
+
+> [!Tip]
+> You can find out the correct `.container` file by copying it from the corresponding tag of the https://github.com/gardenlinux/repo branch, for example [1443.0](https://github.com/gardenlinux/repo/blob/1443.0/.container)
+
+
+> [!Note]
+> We must create rel- branches to include the respective `.container` file, and not use maint for backports. This is required because multiple releases can use the same kernel version (e.g. rel-1443 and rel-1592 both use maint-6.6) 
+
+## Automated kernel patch level upgrades 
+
+A scheduled workflow scans a list of configured branches [see](https://github.com/gardenlinux/package-linux/blob/main/.github/workflows/pr-if-new-kernel.yml#L12), and bumps the patchlevel of the version defined in the prepare_source file.
+The automation creates a PR if a new patchlevel is available.
```

Comment:
```suggestion
The automation creates a PR if a new patchlevel is available.

> [!Important]  
> Note that build failures in this PR will not be visible in the way you are used to it.
> This is due to limitations on GitHub.
> Always check the PR-related workflow manually before merge as it might well be that an upgrade of the kernel breaks the build.
> [See this issue for more information if you are interested](https://github.com/gardenlinux/package-linux/issues/47).

```